src/connector/sql/sqlGeneration.ts

```diff
@@ -382,30 +382,30 @@ export const scalarComparisonOperatorMappings: ScalarOperatorMappings = {
 export function getScalarType(column: Column): string {
   if (column.nestedField) {
     return getNestedScalarType(column.nestedField);
+  }
+
+  if (column.type === undefined) {
```

Comment:
Have `if (!column.type)` here instead? Takes care of `column.type` being both `null` or `undefined`. (more [here](https://stackoverflow.com/a/44017547))

The issue with this is, it's strict checking which means this case is checking only for `undefined` and not `null`. If you want to keep the check explicit, maybe use `if (column.type == null)` (notice only 2 `=` signs instead of three). More about this [here](https://stackoverflow.com/a/28984306)

Reply:
Got it, thanks, I didn't know about this. I will make the change.